### ISSUE TITLE
KRONA: link to "updateAccessions.sh" needs to be placed in PATH during install

### ISSUE THREAD

**Issue by jvollme** [bioconda/bioconda-recipes#10609]
Issue #1527 already dealt with placing a link to "updateTaxonomy.sh" in PATH during installation (renaming it as "ktUpdateTaxonomy" to avoid name-conflicts. 

However, the same needs to be done for "updateAccessions.sh", which is needed to get reliable Taxonomy information from e.g. BLAST results (perhaps renaming the link as "ktUpdateAccesions"?

**Comment by yesimon** [comment]
Sounds good. Feel free to submit a PR and I'll merge it.

**Comment by jvollme** [comment]
Hi, thanks. Will try to do so, but may need a little help...

 I created a fork, made the changes locally and started running the recommended tests using the "Circle CI client method" and the "bootstrapping method" described in the  [bioconda guidelines](https://bioconda.github.io/contribute-a-recipe.html). 

I do not know exactly what behavior to expect from these tests (sorry not a trained Informatician here), but i expect they should all pass without errors before i sould push my changes and submit a pull-request, right?

However at the second step running  ```bioconda-utils lint recipes config.yml --git-range master``` gives me the following error:
```18:21:59 BIOCONDA INFO Recipes newly unblacklisted:

18:21:59 BIOCONDA INFO Recipes to consider according to git: 
recipes/krona/2.5
 recipes/krona/2.5
 recipes/krona/2.6
 recipes/krona/2.6
 recipes/krona
 recipes/krona
18:21:59 BIOCONDA INFO Recipes to lint:
recipes/krona
 recipes/krona/2.5
 recipes/krona/2.6
Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
Multiple meta.yaml files found. The meta.yaml file in the base directory will be used.
Adding in variants from internal_defaults
Adding in variants from /tmp/miniconda/miniconda/conda_build_config.yaml
Adding in variants from /tmp/miniconda/miniconda/lib/python3.6/site-packages/bioconda_utils/bioconda_utils-conda_build_config.yaml
18:22:01 BIOCONDA ERROR 

The following recipes failed linting. See https://bioconda.github.io/linting.html for details:

                                              failed_tests
recipe                                                    
recipes/krona                          [should_not_use_fn]
recipes/krona/2.5  [uses_perl_threaded, should_not_use_fn]
recipes/krona/2.6  [uses_perl_threaded, should_not_use_fn]
```

How can I fix this error, and how should i proceed?

**Comment by yesimon** [comment]
The recipe should be migrated to conda3 with the removal of the `fn` section in the `source` section of `meta.yaml`. Perhaps your fork is out of date? A git pull should resolve that.

**Comment by jvollme** [comment]
Well the fork is new (created just now), but I forgot that I am using conda2 (silly really, but i am used to pyhton 2.7 and having that as the defaut python seemed good at the time). 
So in order to create, modify and test recipes i should be using conda3?

**Comment by yesimon** [comment]
Yes, bioconda recently migrated to conda 3 build. You can still use python2.7 in your personal conda environment. conda 3 is only used for building packages.

**Comment by jvollme** [comment]
OK, thanks. I'll switch to conda3 and retry (not today though). BTW on a related note: is there any chance i can just export/import my existing environments from conda2 (so i can just keep on working on my projects seamlessly)?

**Comment by yesimon** [comment]
You don't need to do anything to maintain working on your projects. `conda-build` 3 is used for building packages. `Anaconda` and `miniconda` are package managers which maintain your packages in your projects - they don't need `conda-build` at all if you're just using packages.
